src/screens/Setup.svelte

```diff
@@ -115,13 +115,13 @@
 				} else if (code == "I:011 | Invalid Password") {
 					loginStatus = "Invalid password!";
 				} else if (code == "E:018 | Account Banned") {
-					loginStatus = "This account is banned. L :(";
+					loginStatus = "This account is banned. LOL :(";
 				} else if (code == "E:107 | Packet too large") {
-					loginStatus = "The username and/or password is too long!";
+					loginStatus = "The username and/or password is too long! Sadly";
 				} else if (code == "E:019 | Illegal characters detected") {
 					loginStatus = "Usernames must not have spaces or other special characters!";
 				} else if (code == "E:106 | Too many requests") {
-					loginStatus = "Too many requests! Please try again later.";
+					loginStatus = "Too many requests! Please try again later. And While your at it blame Us for it";
```

Comment:
BIt of grammar and capitalization error. It should be, "And while you're at it, blame us for it."
But still don't think it really fits with the error.